### DISCUSSION TITLE
Is there anyway to have an ARM64 VM running on a AMD64 machine using cockpit?

### DISCUSSION THREAD

**Discussion by aggarcia42** [cockpit-project/cockpit-machines#942] [Q&A]
I was able to run a arm64 VM using qemu on my amd64 machine.

And now I was in search of a web interface that allows me to control amd and arm machines.

But when I try to execute the arm64 machine on my amd64 machine running cockpit. I keep getting errors.

My question here after a lot of research, is it possible?

**Comment by KKoukiou** [comment]
Theoretically it is possible, but we currently don't test it. I will try to reproduce this week and get back here with more details.
If you can mention which ISOs or other installation media you use, it would be useful. Thanks

**Reply by aggarcia42** [reply]
I am running Cockpit on ubuntu 22.04.1 LTS and trying to emulate raspios_arm64 (actually it can be any arm ISO) but I am trying to emulate a raspberry.
Here I download the raspios.img: https://www.raspberrypi.com/software/operating-systems/
You can have the .iso by using this command:
`genisoimage -o raspiarm64.iso -input-charset utf8 raspiarm64.img `

(I can not call myself an sysadmin expert but based on my research this is how you do it)
